jps-plugin/src/org/jetbrains/jps/incremental/scala/local/CompilerFactoryImpl.scala

```diff
@@ -98,7 +97,9 @@ object CompilerFactoryImpl {
 
     val scalaVersion = scalaInstance.actualVersion
     val sourceJar =
-      if (isBefore_2_11(scalaVersion)) sourceJars._2_10
+      if (scalaVersion.contains("hydra"))
+        scalaInstance.otherJars().find(_.getName.contains("hydra-bridge")).get
```

Comment:
`.get` can throw, use `headOption` and LOG if the hydra-bridge can't be found. In the case the hydra-bridge is missing we should tell the user about the problem

Reply:
What if the Scala version is 2.10, what happens in this case? (we don't have a hydra-bridge for 2.10)

Reply:
Also, the bridge name is `hydra-bridge_1_0` (please extract a constant and let's refer to the constant everywhere)

Reply:
If the Scala Version is 2.10 or something that is not supported, the `scalaVersions` will not contain "hydra" in it so the `scalaVersion.contains("hydra")` condition will be false.

Reply:
Ok. Please log this fact.